mysql-test/suite/innodb/t/xtradb_compressed_columns_upgrade-master.opt

```diff
@@ -0,0 +1 @@
+--loose-compressed-columns-unknown1
```

Comment:
Would --force-restart here achieve the same effect here (and would also help with the next comment)?